lib/Doctrine/Query.php

```diff
@@ -45,16 +55,6 @@
  *              from this tedious task.
  *              This would also largely reduce the currently huge interface of Doctrine_Query(_Abstract)
  *              and better hide all these transformation internals from the public Query API.
- *
- * @internal    The lifecycle of a Query object is the following:
```

Comment:
Psalm yells about these `@internal` annotations in other projects because it interprets them as meaning "this class is for internal use only" rather than the "this comment is for developers but shouldn't be exposed in the documentation".